### PR TITLE
kubeadm: add a presubmit e2e for k/kubeadm/kinder

### PR DESCRIPTION
**Pull request by neolit123**
/hold
for https://github.com/kubernetes/kubeadm/pull/1904
to merge

/assign @fabriziopandini 
/priority important-longterm
/kind feature

### CODE REVIEW

File: config/jobs/kubernetes/kubeadm/kubeadm-presubmits.yaml [hunk 1]
```diff
@@ -10,6 +12,43 @@ presubmits:
       - image: gcr.io/k8s-testimages/kubekins-e2e:v20191108-9467d02-master
         command:
         - "./kinder/hack/verify-all.sh"
+
+  - name: pull-kubeadm-kinder-upgrade-beta-latest
```
Comment (fabriziopandini) [inline config/jobs/kubernetes/kubeadm/kubeadm-presubmits.yaml]: beta-master (see my comments on 1904)

**Comment by neolit123** [comment]
updated.

**Pull request merged**